samples/python_interop/cirq_samples/intro.ipynb

```diff
@@ -0,0 +1,235 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "0352b5aa",
+   "metadata": {},
+   "source": [
+    "# Cirq Introduction: Quantum Random Number Generator\n",
+    "\n",
+    "This notebook introduces basic Cirq concepts: qubits, gates vs. operations, circuits, simulation, and collecting measurement statistics. We'll build a simple quantum random number generator (QRNG) using a Hadamard gate and a measurement."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "dd2cf71f",
+   "metadata": {},
+   "source": [
+    "## 1. Environment Setup\n",
+    "If Cirq isn't installed in your environment you can uncomment and run the next cell. (It's often pre-installed in curated quantum environments.)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "21bc9584",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Optional install (uncomment if needed)\n",
+    "# %pip install cirq --quiet"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c1428ea5",
+   "metadata": {},
+   "source": [
+    "## 2. Imports\n",
+    "We first import Cirq."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "22020141",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import cirq"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e24a71f4",
+   "metadata": {},
+   "source": [
+    "## 3. Defining the QRNG Program\n",
+    "We create a function returning a sequence of Cirq operations. Note the distinction: a *Gate* is a template; an *Operation* applies that gate to specific qubits."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fbc500f4",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from typing import Sequence\n",
+    "\n",
+    "def qrng() -> Sequence[cirq.Operation]:\n",
+    "    a = cirq.NamedQubit(\"a\")  # Could also use LineQubit, GridQubit, etc.\n",
+    "    return [\n",
+    "        cirq.H.on(a),            # Put |0> into equal superposition |+>\n",
+    "        cirq.measure(a, key='result')  # Measure in Z basis -> random 0/1\n",
+    "    ]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "52049f16",
+   "metadata": {},
+   "source": [
+    "## 4. Building a Circuit\n",
+    "A `cirq.Circuit` organizes moments (time steps). We'll print the raw operations and the formatted circuit."
```

Comment:
Suggest mentioning that automatic scheduling is used (vs. manual creation of moments).